questionnaire/questionnaireUISchema.js

```diff
@@ -523,5 +560,54 @@ module.exports = {
             buttonText: 'Agree and submit',
             pageContext: 'submission'
         }
+    },
+    'p-applicant-select-treatments': {
+        // transformer: 'form',
+        options: {
+            transformOrder: [
+                'q-applicant-other-treatment-dmi',
+                'q-applicant-select-treatments-dmi'
+            ],
+            outputOrder: ['q-applicant-select-treatments-dmi'],
+            properties: {
+                'q-applicant-select-treatments-dmi': {
+                    options: {
+                        conditionalComponentMap: [
+                            {
+                                itemValue: 'other',
+                                componentIds: ['q-applicant-other-treatment-dmi']
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+    },
+    'p-gp-enter-your-address': {
+        options: {
+            properties: {
+                'q-gp-building-and-street': {
+                    options: {
+                        macroOptions: {
+                            classes: ''
+                        }
+                    }
+                },
+                'q-gp-building-and-street-2': {
```

Comment:
I *think* these should be `q-gp-building-and-street2` @BarryPiccinni can you review and confirm please